exdb/templates/exdb/experience_view.html

```diff
@@ -13,8 +13,8 @@
 
 {% block content %}
     <div class="row">
-        <div class="small-3 small-centered columns">
-            <h1>{{ experience.name }}</h1>
+        <div class="small-12 small-centered columns">
+            <h1 class="center">{{ experience.name }}</h1>
```

Comment:
Foundation has a helper class "text-center" that does the same thing as this class.